### PR TITLE
Django under the hood

### PR DESCRIPTION
**Pull request by lgh2**
Adds videos for Django Under the Hood 2014 and Django Under the Hood 2016

### CODE REVIEW

File: django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (redapple) [inline django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json]: Missing `{`

Comment (lgh2) [inline django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json]: Thank you! Was there a test that showed the missing curly brace?

Comment (redapple) [inline django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json]: @lgh2 , not currently in the code base, but locally I'm using https://github.com/pyvideo/data/pull/293 to detect this, along with verbose tests `make VERBOSE=1 test`

Comment (redapple) [inline django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json]: In Travis, this showed
```
python3 /home/travis/build/pyvideo/data/.tests/schemas.py -d /home/travis/build/pyvideo/data -s /home/travis/build/pyvideo/data/.schemas -v 0
Error JSON-decoding /home/travis/build/pyvideo/data/django-under-the-hood-2016/videos/nadia-eghbal-about-oss-funding-at-django-under-the-hood-2016.json
make: *** [test-schemas] Error 1
```
See https://travis-ci.org/pyvideo/data/builds/245720689

**Pull request merged**